app/src/test/java/com/example/randomusergenerator/data/repository/UserRepositoryImplTest.kt

```diff
@@ -54,10 +54,13 @@ class UserRepositoryImplTest {
         whenever(mockUserDao.updateUser(any())).thenReturn(Unit)
 
         // WHEN
-        serviceUnderTest.getAllUsers(numberOfUsers)
+        val resultFlow = serviceUnderTest.getAllUsers(numberOfUsers)
+        val result = mutableListOf<Resource<List<UserData>>>()
+        resultFlow.collect {
```

Comment:
@Ruthy4 I don't think it's recommended to use **Flow** operators like `collect{}` in unit tests. Here are a few resources that could help you understand better unit-testing with Turbine: [1](https://medium.com/@wambuinjumbi/unit-testing-flows-37c904ad453), [2](https://levelup.gitconnected.com/unit-test-with-kotlin-flow-7e6f675e5b14), [3](https://medium.com/google-developer-experts/unit-testing-kotlin-flow-76ea5f4282c5).